vaxrank/report.py

```diff
@@ -106,6 +108,11 @@ def _patient_info(self):
             ('Total number of somatic variants', self.patient_info.num_somatic_variants),
             ('Somatic variants with predicted coding effects',
                 self.patient_info.num_coding_effect_variants),
+            ('Somatic variants with predicted coding effects and RNA support',
+                self.patient_info.num_variants_with_rna_support),
+            ('Somatic variants with predicted coding effects, RNA support and predicted MHC '
+                'expression',
```

Comment:
"MHC expression" == there are MHC molecules being made in the cell. 

Maybe use "predicted MHC ligands"?

Reply:
Done